### ISSUE TITLE
Annoying new warning in terminal

### ISSUE THREAD

**Issue by jakobnissen** [RasmussenLab/vamb#341]
When launching new versions of Vamb, this warning appears:
```
/home/jakni/miniconda3/envs/recluster/lib/python3.12/site-packages/numpy/core/getlimits.py:549: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/jakni/miniconda3/envs/recluster/lib/python3.12/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float64'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
/home/jakni/miniconda3/envs/recluster/lib/python3.12/site-packages/numpy/core/getlimits.py:549: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/home/jakni/miniconda3/envs/recluster/lib/python3.12/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
```

As far as I can tell from googling around this happens because
* Some library loaded by Vamb is compiled with `-ffast-math`
* The `-ffast-math` option toggles the FTZ and DAZ flags in the CPU for the duration the library is running. These flags modify the behaviour of floating point ops, _for all libraries in the same process_.
* This may cause incorrect computation

The solution is to figure out which library does this and remove/upgrade/downgrade it.

**Comment by jakobnissen** [comment]
This is due to either pyhmmer or pyrodigal

**Comment by jakobnissen** [comment]
Closed by a temporary fix in #357
